<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qu::ProgressWidget</class>
 <widget class="QWidget" name="qu::ProgressWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>25</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="operationNameLabel"/>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="format">
      <string>%p %</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="pauseButton">
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="cancelButton"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pauseButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>qu::ProgressWidget</receiver>
   <slot>pause(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>qu::ProgressWidget</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pause(bool)</slot>
  <slot>cancel()</slot>
 </slots>
</ui>
